app/services/scenes/scenes.ts

```diff
@@ -1,4 +1,6 @@
 import Vue from 'vue';
+import { uniqBy } from 'lodash';
+import { Subject } from 'rxjs';
```

Comment:
Unused import

Reply:
Which one exactly? Both `uniqBy` and `Subject` are used.